candi/cloud-providers/yandex/openapi/cluster_configuration.yaml

```diff
@@ -124,8 +124,15 @@ apiVersions:
                 type: integer
                 description: |
                   Yandex Compute Instance disk size in gibibytes.
-                example: 20
+                example: network-ssd
                 x-doc-default: 50
+              diskType: &instanceClassDiskType
+                type: string
+                description: |
+                  Yandex Compute Instance disk type.
+                example: network-ssd-io-m3
+                x-doc-default: network-ssd
+                enum: [ network-ssd, network-ssd-io-m3 ]
```

Comment:
may be we need to add all disk types there ?

Reply:
We should not allow HDD to be used. The remaining ssd network-ssd-nonreplicated is added to the enum.